src/frontends/onnx/frontend/src/op/trilu.cpp

```diff
@@ -25,9 +25,18 @@ OutputVector trilu(const Node& node) {
     if (rank.is_static()) {
         CHECK_VALID_NODE(node, rank.get_length() >= 2, "Trilu first input's rank must be >= 2");
     }
+
+    Output<ngraph::Node> k;
```

Comment:
please use `ov` instead of `ngraph` namespace. `ngraph` is deprecated and is going to be removed in the upcoming release.